{"playernames": ["HunterBot.py", "LeftyBot.py", "HunterBot.py", "GreedyBot.py"], "error": "Traceback (most recent call last):\n  File \"d:\\personal\\aichallenge-ants\\tools-linux\\engine.py\", line 92, in run_game\n    sandbox.start(bot_cmd)\n  File \"d:\\personal\\aichallenge-ants\\tools-linux\\sandbox.py\", line 385, in start\n    raise SandboxError('Failed to start {0}'.format(shell_command))\nSandboxError: Failed to start ['python', './HunterBot.py']\n"}